central/main.go

```diff
@@ -364,6 +365,10 @@ func startServices() {
 	gcp.Singleton().Start()
 	administrationEventHandler.Singleton().Start()
 
+	if features.PlatformComponents.Enabled() {
```

Comment:
What happens to the regular queries when this is turned off?

Reply:
What does regular queries mean? I can gate the changes in alert and deployment datastores that set the platform component field.